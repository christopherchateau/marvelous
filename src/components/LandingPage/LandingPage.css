.Main {
  background-color: black;
}

.img-wrapper {
  display: flex;
  justify-content: center;
  align-content: center;
  height: 100%
}

.marvel-uni-img {
  background-color: grey;
}