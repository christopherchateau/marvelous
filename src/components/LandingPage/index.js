import React from "react";
import { NavLink } from "react-router-dom";
import Header from "../../components/Header";
import marvelUniverse from "../../images/marvel-universe.png";
import "./LandingPage.css";

const LandingPage = () => {
  return (
    <NavLink className="Main" to='/characters'>
    <Header />
      <div className="img-wrapper">
      <h3 className="click-anywhere">Click to begin!</h3>
      </div>
    </NavLink>
  );
};

export default LandingPage;
