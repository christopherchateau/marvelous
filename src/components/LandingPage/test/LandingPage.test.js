import React from "react";
import { shallow } from "enzyme";
import LandingPage from "../index";

describe("LandingPage", () => {
  const wrapper = shallow(<LandingPage />);
  it("should exist", () => {
    expect(wrapper).toBeDefined;
  });

  it("should render like snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});