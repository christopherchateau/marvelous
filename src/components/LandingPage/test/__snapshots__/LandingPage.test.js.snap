// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should render like snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="img-wrapper"
      >
        <img
          alt="Marvel Universe"
          className="marvel-uni-img"
          src="marvel-universe.png"
        />
      </div>,
      "className": "Main",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="Marvel Universe"
          className="marvel-uni-img"
          src="marvel-universe.png"
        />,
        "className": "img-wrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Marvel Universe",
          "className": "marvel-uni-img",
          "src": "marvel-universe.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="img-wrapper"
        >
          <img
            alt="Marvel Universe"
            className="marvel-uni-img"
            src="marvel-universe.png"
          />
        </div>,
        "className": "Main",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="Marvel Universe"
            className="marvel-uni-img"
            src="marvel-universe.png"
          />,
          "className": "img-wrapper",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Marvel Universe",
            "className": "marvel-uni-img",
            "src": "marvel-universe.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "div",
      },
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
